activities/communication/communications-plan.md

```diff
@@ -0,0 +1,169 @@
+
+Messages:
+
+* Find out more about [place or institution’s] interesting public code(ish) approach/project
+* Find out more about the Foundation for Public Code and its work (new member signed up, new codebase incubated)
+* Find out more about the Foundation for Public Code its vision
+* International movements/partnerships for being smarter about technology/digital infrastructure (“you want to be part of this club of innovators”)
+* Find out more about the Standard and certification (how we contribute to risk reduction)
+* Find out more about codebase reuse (multiplying tech investment or saving money)
+
+### Users and contributors
+
+This includes:
+
+* the policy community
+* [the tech and open source communities](../value-and-impact/user-mapping/developers-and-designers.html)
+* [business management and procurement for public administrations](../value-and-impact/user-mapping/management-and-procurement.html)
+
+Messages:
+
+* Find out more about [place or institution’s] interesting technological approach to public code or open source (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about [place or institution’s] interesting policy approach to public code or open policymaking (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about [place or institution’s] interesting management approach to procuring or producing public code or open source code (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about the Standard and our other resources for different user groups
+* Come to meetups and events
+* News from and profiles of our contributor community (‘join our thriving community!’)
+
+### Other interested parties
+
+This includes:
+
+* vendors and the private sector
+* end users (residents and citizens)
+* journalists, academia and non-governmental organizations
+
+Messages:
+
+* Good news from our partners and affiliates (eg, relevant session at organization X’s annual conference!)
+* Any news about Foundation or codebase impact for end users (eg, check out these communications by a member about their new software for managing parking)
+* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
+* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))
+
+## Channels
+
+Our channels are:
+
+* [Our blog](https://blog.publiccode.net/): long form, platform of record (read more about our blog)
+* [Twitter](https://twitter.com/publiccodenet/): short form, ephemera
+* LinkedIn: share our blogposts or reshare others' content about us, ephemera (mostly avoid, since it’s a closed platform)
```

Comment:
```suggestion
* [LinkedIn](https://www.linkedin.com/company/publiccodenet/): share our blogposts or re-share others' content about us, ephemera (mostly avoid, since it’s a closed platform)
```